### DISCUSSION TITLE
text align method

### DISCUSSION THREAD

**Discussion by prodan-eric** [josdejong/svelte-jsoneditor#200] [Q&A] [answered]
is there a method on the class  to access the same functionality the format button on the main menu uses?

**Comment by josdejong** [comment, marked as answer]
No, but the code boils down to:

```js
function format(jsonText, indentation = 2) {
  return JSON.stringify(JSON.parse(jsonText), null, indentation)
}
```
Does that help? Note that `JSON.parse` can throw an exception when the text contents is not valid JSON.
